src/components/Apps/AccountCreateUpdateForm/index.vue

```diff
@@ -63,7 +63,7 @@ export default {
       encryptedFields: ['secret'],
       fields: [
         [this.$t('Basic'), ['name', 'username', 'privileged', 'su_from', 'su_from_username', 'template']],
-        [this.$t('Asset'), ['assets']],
+        [this.$t('Asset'), ['nodes', 'assets']],
         [this.$t('Secret'), [
           'secret_type', 'password', 'ssh_key', 'token',
           'access_key', 'passphrase', 'api_key',
```

Comment:
The first code snippet contains several mistakes that you need to fix before using it. The correct way is:

```json
{
  "export": true,
  "dependencies": {}
}
```

and the corrected second snippet should look like:

```javascript
[[
  this.i18n.t("Basic"),
  ["name", "username"],
  ["privileged", "su_from", "su_from_username", "template"]
]+

[[this.i18n.t ("Asset"), [
   "nodes",
   "assets"
]]]

[
  this.i18n.t("Secret"),
  [
    "secret_type", "password",
    "ssh_key",
    "token",
    "access_key",
    "passphrase",
    "api_key"]

  ]
]
```